### ISSUE TITLE
Clecio's NRM ES dashboard moved to core accounts

### ISSUE THREAD

**Issue by ActionAnalytics** [bcgov/cloud-pathfinder#852]
**Describe the issue**
NRM is paying for the ES, we want to move it to core accounts to take that burden off them

**Which Sprint Goal is this issue related to?**
> Note: 'Milestone' is a ZenHub term that we use synonymously with 'Sprint'. 

The 'Milestone' description (created in ZenHub) should clearly list Sprint Goals. This section should indicate which 'goal' this issue is related to.

**Additional context**
- ES = Elastic Search

**Definition of done**
- Clecio needs to be able to interface with ES front end
- Consult with Clecio about how to handle Dev/Test/Prod
-

**Comment by sheaphillips** [comment]
Consulted with NRM OneTeam Tech lead for additional background on the need described in this ticket. The team is taking the approach of building a PoC over the next several months and has made arrangement to avoid/reduce costs so shuffling their resources around to reduce cost is not a priority from their perspective. Consulted CPF PO and concluded that this ticket could be closed and perhaps the concern re-evaluated in ~6 months once the PoC is complete or further along.
